aten/src/ATen/core/function_schema.h

```diff
@@ -602,4 +607,15 @@ inline std::string toString(const FunctionSchema& schema) {
 
 } // namespace c10
 
+namespace std {
+template<>
+  struct hash<c10::SchemaArgument> {
+    size_t operator()(const c10::SchemaArgument& arg) const
+    {
+      return std::hash<std::string>()(std::to_string((int)(arg.type)) + ":" + std::to_string(arg.index));
```

Comment:
Hash the values directly instead of converting the values to strings.

Reply:
Wouldn't I need to combine the hashes then? I thought that converting the values to a string would be simpler then messing with combining the hashes of type and index, but I might be mistaken about this.

Reply:
https://github.com/pytorch/pytorch/blob/master/c10/util/hash.h#L39-L41